view/src/main/webapp/WEB-INF/jsp/proposals/contestProposals/header_contest_details.jspx

```diff
@@ -175,6 +177,37 @@
                         </a>
                     </c:if>
                 </div>
+
+                <c:if test="${showJudgingStatus}">
```

Comment:
@jobachhu this one was already available from contestProposals.jspx, i guess it is best to reuse in this case. Missed this one earlier.